ldap/servers/slapd/log.c

```diff
@@ -224,64 +217,41 @@ slapd_log_error_proc_internal(
     va_list ap_err,
     va_list ap_file);
 
-/*
- * these macros are used for opening a log file, closing a log file, and
- * writing out to a log file.  we have to do this because currently NSPR
- * is extremely under-performant on NT, while fopen/fwrite fail on several
- * unix platforms if there are more than 128 files open.
- *
- * LOG_OPEN_APPEND(fd, filename, mode) returns true if successful.  'fd' should
- *    be of type LOGFD (check log.h).  the file is open for appending to.
- * LOG_OPEN_WRITE(fd, filename, mode) is the same but truncates the file and
- *    starts writing at the beginning of the file.
- * LOG_WRITE(fd, buffer, size, headersize) writes into a LOGFD
- * LOG_WRITE_NOW(fd, buffer, size, headersize, err) writes into a LOGFD and
- *  flushes the buffer if necessary
- * LOG_CLOSE(fd) closes the logfile
```

Comment:
I think it's worth leaving at least some of the comments and putting them around the new functions.

Reply:
> I think it's worth leaving at least some of the comments and putting them around the new functions.

There were not many comments about LOG_WRITE* but I added them back